### PR TITLE
[12.0][MIG] project_timesheet_time_control - alternative

### PR DESCRIPTION
**Pull request by alexey-pelykh**
https://github.com/OCA/hr-timesheet/pull/179, https://github.com/OCA/hr-timesheet/pull/180, and this one should provide a seamless drop-in substitution to #447. Also depends on https://github.com/OCA/hr-timesheet/pull/139

### CODE REVIEW

File: project_timesheet_time_control/models/account_analytic_line.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+# Copyright 2019 Brainbean Apps (https://brainbeanapps.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class AccountAnalyticLine(models.Model):
+    _inherit = 'account.analytic.line'
+
+    # defines start date, start date + unit amount = end date
```
Comment (alexey-pelykh) [inline project_timesheet_time_control/models/account_analytic_line.py]: @pedrobaeza I'll need your help over here

**Review by yajo — CHANGES_REQUESTED** [review]
This module deletes the previous implementation to just add a pre-migration script. AFAICS, you should install the module in an updated database and then uninstall it once migration is passed.

If this one were the path to follow, doing it in a module makes no sense. It should be done in OpenUpgrade instead.

Sorry, :man_shrugging: but :-1:, IMHO this can be closed.

**Comment by alexey-pelykh** [comment]
@Yajo thanks for the feedback! Probably OpenUpgrade is more proper way

**Comment by yajo** [comment]
Can you close this then, please? I don't have permissions.

Thanks for your understanding! :wink:

**Comment by alexey-pelykh** [comment]
Once there will be a code to reference as "superseded by" - sure

**Pull request closed without merging**